optimum_benchmark/backends/onnxruntime/backend.py

```diff
@@ -316,7 +316,6 @@ def prepare_for_profiling(self, input_names: List[str]) -> None:
         LOGGER.info("\t+ Wrapping model inside profiler")
         self.pretrained_model = ORTProfilingWrapper(self.pretrained_model)
 
-    @record
```

Comment:
Probably there should be an other `train` method with DDP then?

Reply:
we'll still need to import `record`, what's problematic is that `pickle doesn't accept wrapped functions. 
I think the only way to keep it is to have an -ugly- if else with a dummy wrapper to use instead. wdyt ?

Reply:
ah or else we wrap the wrapper 😆